lib/matplotlib/dates.py

```diff
@@ -1013,10 +1019,13 @@ def __init__(self, base=1, month=1, day=1, tz=None):
 
     def __call__(self):
         dmin, dmax = self.viewlim_to_dt()
```

Comment:
This should probably get the same try/except logic that the baseclass and `MicrosecondLocator` have.